locations/templates/quarkus-application/template.yaml

```diff
@@ -330,13 +330,15 @@ spec:
           git_branch: main
           image_url: ${{ parameters.imageUrl }}
           appNamespace: ${{ parameters.namespace }}
+          version: ${{ parameters.version }}
+          quarkusVersion: ${{ parameters.quarkusVersion | extractVersionFromKey }}
```

Comment:
We should remove the `| extractVersionFromKey` here as the `parameters.quarkusVersion` includes now the `major.minor.patch` version and not anymore `io.quarkus.platform:major.minor`

Reply:
Yes, I missed this one